Design.md

```diff
@@ -229,6 +231,20 @@ This API will enable or disable user on Vault.
 ### Create S3 Credential
 This API creates S3 credential for the user.
 1. `generate-access-key` api will be called using assumed role credentials.
+1. Encrypt the `secret-key` using AES algorithm and store the encrypted `secret-key` on Redis Cache in the hash named as `OSIS` with the hash-key formatted as `<Username>__<AccessKeyID>`.
+1. Log the `access-key-id` _(Log location and tools for logging are yet to be decided)_
```

Comment:
Either: 
Log `access-key-id`
or 
Log the access key ID

Reply:
```suggestion
1. Log the access key ID _(Log location and tools for logging are yet to be decided)_
```